src/Components/Components/src/Microsoft.AspNetCore.Components.netcoreapp.nuspec

```diff
@@ -3,7 +3,7 @@
   <metadata>
     $CommonMetadataElements$
     <dependencies>
-      <group targetFramework=".net5.0">
+      <group targetFramework=".net6.0">
```

Comment:
It's pretty easy to extend the properties passed into `*.nuspec` files. Should do that to avoid having to change this again (relatively soon).